app/assets/stylesheets/provider/_links.scss

```diff
@@ -16,11 +19,10 @@ a {
   }
 }
 
-@mixin iconic($content) {
-  font-family: FontAwesome;
-  font-weight: $font-weight-normal;
+@mixin iconic($content, $font-weight: $font-weight-normal) {
+  font-family: "Font Awesome 5 Free";
```

Comment:
After removing `font-awesome-rails` we're now using version 5 of font-awesome, which requires a font-family update.